website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -4,19 +4,72 @@ id: "user-tokens"
 pagination_next: "docs/dbt-cloud-apis/service-tokens"
 ---
 
+:::note Action required
+
+The [user API tokens](#user-tokens) will eventually be deprecated. The deprecation date is yet to be determined, but we recommend you update to account-scoped personal access tokens to avoid service disruptions in the future. We will communicate, with ample notice, the deprecation date when it has been determined.
+
+The current API key is located under **Personal Settings → API Key.** 
+
+Please [contact support](/docs/dbt-support#dbt-cloud-support) with any questions or concerns.
+
+:::
+
+## Account-scoped personal access tokens
+
+:::info New 
+
+On Feb 7, 2024, we introduced a new type of token for individual users called personal access tokens. Note that these differ from [Service Tokens or API Keys](/docs/dbt-cloud-apis/authentication#types-of-api-access-tokens). Before this release, user API keys were the only way to access dbt Cloud API on behalf of the user. These API Keys were user-specific and were not scoped to an account. To enhance the security of dbt Cloud, we are moving away from this model to account-specific tokens. 
```

Comment:
Does this need to be a note? Feels like a 1st paragraph?

Reply:
I've worked under the assumption that anything with dates should be temporary text, but I'm happy to change it.